src/snippets/meta.html

```diff
@@ -0,0 +1,17 @@
+<title>Brenna's Food Software</title>
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script
+    async
+    src="https://www.googletagmanager.com/gtag/js?id=G-H504GHJSBG"></script>
+<script>
+    window.dataLayer = window.dataLayer || [];
+
+    function gtag() {
+        dataLayer.push(arguments);
+    }
+
+    gtag('js', new Date());
+    gtag('config', 'G-H504GHJSBG', {
+        custom_map: { dimension1: 'uid' },
+    });
+</script>
```

Comment:
There's an IntelliJ checkbox to ensure files always end with a newline at the bottom of Settings > Editor > General. No worries here, but for the future.